systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Sign Out" %}{% endblock %}
+{% block content %}
+  <div class="mt40"></div>
+  <div class="container">
+    <form class="logout form-logout" role="form" method="post" action="{% url 'account_logout' %}">
+      <h2>{% trans "Sign Out" %}</h2>
+      <hr/>
+      <p>{% trans 'Are you sure you want to sign out?' %}</p>
+      {% csrf_token %}
+      {% if redirect_field_value %}
+        <input type="hidden" name="{{redirect_field_name}}" value="{{redirect_field_value}}"/>
+      {% endif %}
+      <div class="mt40"></div>
```

Comment:
I just tested the form without the space between the message and the button, and it looks just fine.
![image](https://cloud.githubusercontent.com/assets/2039122/6057162/5ea15772-ad4d-11e4-8a14-334861bb13dc.png)